src/Lib/Helpers/ArrayHelper.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Oleksandr Rokytskyi
+ * Date: 06.11.2018
+ * Time: 19:17
+ */
+
+namespace SphereMall\MS\Lib\Helpers;
+
+/**
+ * Class ArrayHelper
+ * @package SphereMall\MS\Lib\Helpers
+ */
+class ArrayHelper
+{
+    /**
+     * @param $data
+     * @param bool $recursive
+     * @return array
```

Comment:
If we put string 'abc' or integer value we won't get an array as result. So:

```suggestion
     * @return mixed|array
```